### DISCUSSION TITLE
GestureEvents.gestureAttached for Android

### DISCUSSION THREAD

**Discussion by ghost** [NativeScript/NativeScript#9841] [General]
> gestures: GestureEvents.gestureAttached added to modify native recognizers when needed (168a169)

https://github.com/NativeScript/NativeScript/blob/168a16972623109afadedabd32ff22d70555026b/packages/core/ui/gestures/index.android.ts#L333

I think it is `android: recognizer,`

**Comment by farfromrefug** [comment]
@ett-hyoshida good catch. can you create the PR or do you want me to do it?

**Reply by ghost** [reply]
Please do it your side. Thank you.

**Reply by farfromrefug** [reply]
@ett-hyoshida https://github.com/NativeScript/NativeScript/pull/9842
